spec/fixtures/files/profile_fields.csv

```diff
@@ -0,0 +1,4 @@
+Name,text_field,John Doe,
```

Comment:
Example CSV used in the spec. Includes a blank line (which will be skipped) as well as a correctly quoted string with comma inside.